chutney/action-impl/src/main/java/com/chutneytesting/action/function/MathFunctions.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * SPDX-FileCopyrightText: 2017-2024 Enedis
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package com.chutneytesting.action.function;
+
+import com.chutneytesting.action.spi.SpelFunction;
+
+public class MathFunctions {
+
+    @SpelFunction
+    public static Number abs(Number a) {
+        String numberType = a.getClass().getSimpleName();
+
+        return switch (numberType) {
```

Comment:
we can extract switch into a private method(`numericValue(Number a)`) and call it then in `abs`, `min`...

Reply:
If I do that numericValue() will return automatically a Number and then it would be impossible to call it in abs, min. These methods don't take Number as input.